orchestrator-salt/src/main/resources/salt/salt/top.sls

```diff
@@ -6,6 +6,9 @@ base:
     - metadata
     - docker
     - recipes.runner
+    {# Can some of these be parallelized? fluentd, metering, ipa seem like good candidates which can work independently.}
```

Comment:
We can look into it, but I expect issues with depending states are not waiting for each other which we can fix.

Reply:
my thoughts: I think we can achieve/win more if we would run some states parallel (salt can do that) - that are not depending hardly on each other (like in my case, telemetry related services) instead of removing checks that can be used to avoid deployment issues. as I see command states itself can be parallel (so you won't need to send multiple salt commands through CB)